packages/nimble-components/tsconfig.json

```diff
@@ -20,7 +20,14 @@
         "importsNotUsedAsValues": "error",
         "noImplicitOverride": true,
         "noUncheckedIndexedAccess": true,
-        "noPropertyAccessFromIndexSignature": true
+        "noPropertyAccessFromIndexSignature": false,
+        // Necessary to fix build error in storybook code
+        // ../../node_modules/@storybook/html/dist/types-2ec88714.d.ts:2:38 - error TS2307: Cannot find module 'lib/docs-tools/dist' or its corresponding type declarations.
+        // import * as lib_docs_tools_dist from 'lib/docs-tools/dist';
+        // TODO: File bug or try newer rc build
+        // Can also fix locally manually modifying that file to import from @storybook/docs-tools/dist
```

Comment:
We have the infrastructure to accomplish patching, pretty straightforward.
Would add a * dep to the package at root so it gets pulled up and then make the change in node_modules and create the patch https://github.com/ds300/patch-package#making-patches
which will get added to the root patches/

Would be nice to have the issue and patch or just issue if they seem receptive to fixing it.

Just want some way to avoid disabling skipLibCheck for an extended period of time since we are deeply integrating several third party libs with pretty complex typing at this point.